src/styles/index.css

```diff
@@ -0,0 +1,21 @@
+@import "./page.css";
+@import "./card.css";
+@import "./review.css";
+@import "./stars.css";
+@import "./price.css";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #060b35;
```

Comment:
I don't know if it is correct or not, but i am having trouble with the same situation in the catalog. This way the color and in my case font-weight effects everything and i had to remove them and add places where it is needed again. Just a reminder :)

Reply:
Not sure about this one, so gonna pass it to the mentors :)

Reply:
it's a good idea to set the most-used font size and line height and them only change it where you need. so good decision